CMakeLists.txt

```diff
@@ -103,6 +103,7 @@ find_package(GrOsmoSDR)
 find_package(LibHackRF)
 find_package(UHD)
 find_package(OpenSSL REQUIRED)
+find_package(CURL REQUIRED)
```

Comment:
I figure if curl is eventually going to replace the custom HTTP logic, then this should be required. The parent Docker image should be updated though to include the dev headers for curl.